docs/en/ingest-management/elastic-agent/view-elastic-agent-status.asciidoc

```diff
@@ -0,0 +1,76 @@
+[[view-elastic-agent-status]]
+[role="xpack"]
+= View status of {agent}s
+
+To view the status of your {fleet}-managed agents, go to
+*Management > Fleet > Agents*.
+
+[role="screenshot"]
+image::images/agent-status.png[Agents tab showing status of each {agent}]
+
+{fleet} reports these statuses:
+
+|=== 
+
+| *Healthy* | The {agent} is online and running as expected. There are no agent
+policy updates or automatic agent binary updates pending, but there might be
+manual agent binary updates available.
+
+| *Unhealthy* | The {agent} is online, but requires attention because it's
+reporting either agent or integration-level errors.
+
+| *Updating* | The {agent} is online and updating the agent policy, updating the
+binary, or enrolling or unenrolling from {fleet}.
+
+| *Offline* | The {agent} has not checked in during the expected time period.
+
+| *Inactive* | The {agent} is inactive.
+
+|===
+
+//QUESTION: What else can we say about the offline status? That the agent
+//might be stopped or no longer able to connect to Fleet? What else?
+
+//QUESTION: What does inactive mean? It shows up in the dropdown.
+
+//QUESTION: Can users set alerts on these statuses yet? If so, we should mention
+//that here.
```

Comment:
@mostlyjason Can you respond to these questions? I can merge as-is but thought it would be good to add more detail if we can.

Reply:
Inactive means the agent is no longer enrolled in Fleet. This is a terminal state and agents must reenroll if they wish to connect to Fleet again.

Reply:
I'm not aware of any way to set an alert.  I don't think people would want to because presumably they unenrolled the agent because they are done with it.